### ISSUE TITLE
Broken JSON requests

### ISSUE THREAD

**Issue by davidlesches** [kyleboe/zoom_rb#25]
There is something wrong with the #meeting_create method. After trying variations of this over and over -

```
zoom_client.meeting_create(
      :user_id    => 'me',
      :type       => 2,
      :topic      => "My Test Event",
      :start_time => 1.day.from_now,
      :duration   => 60
    )
```

I kept getting a `Request Body should be a valid JSON object.` response. 

After tearing my hair out for two hours, I tried the @eclemens fork and it worked fine. Not sure where the issue is, just wanted to note it.

**Comment by kyleboe** [comment]
Pushed 207d8e9a6880dbc266842175a876e49a3dc2be02 to resolve this. My bad. I'll leave this open to serve as a reminder to myself to write more thorough end-to-end tests.

**Comment by davidlesches** [comment]
Thanks!!!

**Comment by kyleboe** [comment]
Thank you for posting. My hope is that  people PR their forks so that we can all work from a relatively complete API wrapper. I have some work time allocated for this in coming weeks so I'll be sure to get things merged as quickly as possible.

**Comment by blazejkotowski** [comment]
Hey, it's the same story with the roles API that was merged a couple of days ago
https://github.com/hintmedia/zoom_rb/pull/22
I will bring the `to_json` back and create another PR shortly.

**Comment by ueki-hikonuki** [comment]
I believe user.rb has a similar problem.

It should be "body: params.except(:id).to_json" in user_update. Otherwise I got "Request Body should be a valid JSON object."

I didn't test other methods though.

**Comment by kyleboe** [comment]
I believe this has been resolved by the refactoring work done in #392
